packages/at_client/lib/src/manager/at_client_manager.dart

```diff
@@ -74,8 +75,26 @@ class AtClientManager {
   }
 
   void _notifyListeners(SwitchAtSignEvent switchAtSignEvent) {
-    for (var listener in _changeListeners) {
+    List<AtSignChangeListener> copyOfChangeListeners = List.from(_changeListeners);
```

Comment:
Lets add Dart docs for the reason to clone the list

Reply:
Sure Jagan, Will add the comments